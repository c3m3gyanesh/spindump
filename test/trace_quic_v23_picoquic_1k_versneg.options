--format json --report-spin-flips