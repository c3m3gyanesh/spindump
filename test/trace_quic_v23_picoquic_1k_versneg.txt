Picoquic against picoquic, wrong  version number. Should result in version negotiation packet but no further traffic.
