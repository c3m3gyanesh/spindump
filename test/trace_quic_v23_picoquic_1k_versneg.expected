[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "d7cb00c3e2ed0e5008048e-null (60489:4433)", "Ts": 1571079275245683, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "d7cb00c3e2ed0e5008048e-048e044f7adaa8dd08d7cb (60489:4433)", "Ts": 1571079275321719, "State": "Starting", "Right_rtt": 76036, "Packets1": 1, "Packets2": 0, "Bytes1": 1280, "Bytes2": 0, "Bandwidth1": 1280, "Bandwidth2": 0 }
]
